osgbuild/promoter.py

```diff
@@ -82,6 +82,9 @@ def __init__(self, pkg_or_build, dist, reason):
     def __str__(self):
         return self.reason % {'pkg_or_build': self.pkg_or_build, 'dist': self.dist}
 
+    def __lt__(self, other):
+        return (self.pkg_or_build, self.dist, self.reason) < (other.pkg_or_build, other.dist, other.reason)
```

Comment:
This is interesting, does this actually work? Seems weird to compare `reason` across the two instances

Reply:
Yeah this makes `sorted()` work. I just want to sort the output by package name (the other fields are there as tie-breakers).